pkgs/development/tools/godot/4/xfixes.patch

```diff
@@ -1,12 +1,12 @@
 diff --git a/platform/linuxbsd/detect.py b/platform/linuxbsd/detect.py
-index ac69f3806b..14acbf5c58 100644
+index 844b15e9fb..0c8bee1757 100644
 --- a/platform/linuxbsd/detect.py
 +++ b/platform/linuxbsd/detect.py
-@@ -191,6 +191,7 @@ def configure(env: "Environment"):
-         env.ParseConfig("pkg-config xrandr --cflags --libs")
-         env.ParseConfig("pkg-config xrender --cflags --libs")
-         env.ParseConfig("pkg-config xi --cflags --libs")
-+        env.ParseConfig("pkg-config xfixes --cflags --libs")
+@@ -192,6 +192,7 @@ def configure(env: "Environment"):
+         env.ParseConfig("pkg-config xrandr --cflags")
+         env.ParseConfig("pkg-config xrender --cflags")
+         env.ParseConfig("pkg-config xi --cflags")
++        env.ParseConfig("pkg-config xfixes --cflags")
```

Comment:
I wonder why upstream doesn't have this. I'm pretty sure they also depend on xfixes don't they?

Reply:
Yes, looks like `xfixes` comes together with some other `x*` lib on the distros that they use, but I'm not sure, [there's a closed PR about this issue](https://github.com/godotengine/godot/pull/59739).

From what I researched, looks like on nix some `x*` libs do share some others `x*` libs inside themselves, but not `xfixes`

On archlinux `xfixes` is a [dependency of `xcursor`](https://archlinux.org/packages/extra/x86_64/libxcursor/)

Reply:
Yes, looks like `xfixes` comes together with some other `x*` lib on the distros that they use, but I'm not sure, [there's a closed PR about this issue](https://github.com/godotengine/godot/pull/59739).

For example, on archlinux `xfixes` is a [dependency of `xcursor`](https://archlinux.org/packages/extra/x86_64/libxcursor/)

From what I researched, looks like on nix some `x*` libs do share some others `x*` libs inside themselves, but not `xfixes` (at least from the point of view of `pkg-config`, since it can't find the lib)

Other related issues:
- [Nix build error (missing headers)](https://github.com/godotengine/godot/issues/59913)
- [xorg.libXi pkg-config does not include xorg.libXfixes cflags](https://github.com/NixOS/nixpkgs/issues/172748)